client/src/legacy/CMSMain.EditForm.js

```diff
@@ -377,17 +377,65 @@ $.entwine('ss', function($){
 	 * Enable save buttons upon detecting changes to content.
 	 * "changed" class is added by jQuery.changetracker.
 	 */
-	$('.cms-edit-form.changed').entwine({
-		onmatch: function(e) {
-			this.find('button[name=action_save]').button('option', 'showingAlternate', true);
-			this.find('button[name=action_publish]').button('option', 'showingAlternate', true);
+  $('.cms-edit-form.changed').entwine({
+    onmatch: function(e) {
+      var save = this.find('button[name=action_save]');
+
+      if(save.attr('data-text-alternate')) {
+        save.attr('data-text-standard', save.text());
+        save.text(save.attr('data-text-alternate'));
+      }
+
+      if(save.attr('data-btn-alternate')) {
+        save.attr('data-btn-standard', save.attr('class'));
+        save.attr('class', save.attr('data-btn-alternate'));
+      }
+
+
+      save
+        .removeClass('btn-secondary-outline')
+        .addClass('btn-primary');
+
+			var publish = this.find('button[name=action_publish]')
+
+      if(publish.attr('data-text-alternate')) {
+        publish.attr('data-text-standard', publish.attr('data-text-alternate'));
+        publish.text(publish.attr('data-text-alternate'));
+      }
+
+      if(publish.attr('data-btn-alternate')) {
+        publish.attr('data-btn-standard', publish.attr('class'));
+        publish.attr('class', publish.attr('data-btn-alternate'));
+      }
+
+      publish
+        .removeClass('btn-secondary-outline')
+        .addClass('btn-primary');
```

Comment:
Took me a while to realise it was working with the `save` and `publish` button with the same actions... maybe select both in the `.find()` and do a `.forEach()` to reduce the repetition?